pybossa/view/projects.py

```diff
@@ -1436,7 +1433,7 @@ def _get_locks(project_id, task_id):
             project_id)
     locks = sched.get_locks(task_id, timeout)
     now = time.time()
-    lock_ttls = {int(k.replace('.', '')): float(v) - now
```

Comment:
under what circumstance does lock ttls would have decimal point value and going forward, it not have decimal point value anymore?